gambling.txt

```diff
@@ -13,6 +13,14 @@
 #
 # Gambling list
 # ------------------------------------[FILTERS]-------------------------------------
+0.0.0.0 promo.casinoroom.com
+0.0.0.0 kaszinoonline.net
+0.0.0.0 icecasino.com
+0.0.0.0 fezbet.com
+0.0.0.0 lottoland.eu
+0.0.0.0 wazamba.com
```

Comment:
This one too

![image](https://user-images.githubusercontent.com/44526987/184009974-26037103-ecbe-49fb-a5b4-bf8893a638c2.png)